content/docs/source-git/work-with-source-git/pull-upstream-fixes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Pull fixes from the upstream"
+date: 2021-01-26T23:59:59+01:00
+draft: false
+weight: 2
+---
+
+# Pull fixes from the upstream
+
+One of the common tasks of being a downstream maintainer is to pull fixes from
+the upstream codebase. This starts by creating a new bugzilla with an issue in
+the distribution which requires a code-change in the affected package. The
+maintainer needs to pull it from upstream when:
+1. The fix is not released yet by the upstream - this implies pulling the fix
+   from the main development upstream branch.
+2. If the fix is already released, one can either update to that upstream
+   release or only pull commits with the fix. Please consult [Fedora Packaging
+   Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/) if
```

Comment:
Would it be possible to narrow the scope of this link? The Guidelines are pretty long, which section/s is/are relevant here?

Reply:
will do - the relevant section should be "don't change API/ABI in a stable Fedora release)

Reply:
okay, I even linked to the wrong doc, thanks for the pointer!